meta-balena-common/conf/distro/include/balena-os.inc

```diff
@@ -2,6 +2,9 @@
 require conf/distro/poky.conf
 include conf/distro/include/balena-os-yocto-version.inc
 
+# Optimize for size
+FULL_OPTIMIZATION = "-Os -pipe ${DEBUG_FLAGS}"
```

Comment:
@jakogut what's the size reduction we get from these? Could you please get some numbers with the image build history class?

Reply:
@alexgg For x86_64:
```
-IMAGESIZE = 674520
+IMAGESIZE = 640636
```

For aarch64:
```
-IMAGESIZE = 646272
+IMAGESIZE = 612552
```

Reply:
thanks @jakogut.

Reply:
@alexgg I rebuilt the aarch64 image with only the userland space optimizations, excluding the kernel config:
```
-IMAGESIZE = 646272
+IMAGESIZE = 638200
```

So it looks like on aarch64, userland size optimization saves about 8 MB, and kernel size optimization saves another 25 MB, most of which is in kernel modules.

Reply:
thanks @jakogut. I think this is something to take into account for those devices that are tight in space, but not something we would want to change as a distro setting.